sites/public/lib/constants.ts

```diff
@@ -1 +1,6 @@
 export const DATE_FORMAT = "MMMM D, YYYY"
+
+export enum USER_STATUS {
```

Comment:
nit: our other enums are written with camel case

Reply:
Updated to camel case.